app/services/amr/upsert_validated_readings.rb

```diff
@@ -0,0 +1,29 @@
+module Amr
+  class UpsertValidatedReadings
```

Comment:
This has now been separated from the validate and persist structure we had before. This just upserts